api/backend/ecs/utilities.go

```diff
@@ -187,10 +188,6 @@ var GetLogs = func(cloudWatchLogs cloudwatchlogs.Provider, taskARNs []*string, t
 
 	logFiles := []*models.LogFile{}
 	for _, logStream := range logStreams {
-		if *logStream.StoredBytes == 0 {
```

Comment:
I confirmed the issue with v0.9.0. The error stems from FirstEventTimestamp property being accessed without checking for nil. 

```
// since the time range is exclusive, expand the range to get first/last events
logEvents, err := cloudWatchLogs.GetLogEvents(
	config.AWSLogGroupID(),
	*logStream.LogStreamName,
	*logStream.FirstEventTimestamp-1,
	*logStream.LastEventTimestamp+1,
	int64(tail))
```

As this code has been removed in v0.10.0 empty logs will no longer generate the json marshal error in theory (and observed in practice) . Based on this, we can be sure that it won't re-introduce the original error.